docker-compose.yml

```diff
@@ -1,43 +1,52 @@
-version: '3'
+version: '3.1'
+# Enable `worker` & `redis` containers support asycronous workers
+# Enable `stripe` container when procesing payments
+# Enable `localstack` container for aws support
 
 services:
   web: &app_base
+    networks: [rails_starter_network]
     build: .
     ports:
       - "3000:3000"
```

Comment:
We should write a rake task for switching the port you visit locally. 3000 is so commonly used for other projects that it's one of the first things I do. It's also a pain because it requires changes here, in .env, and config/puma.rb at least.

Reply:
Just `docker-compose` and `.env.development` (`config/puma.rb` doesn't change since you still serve the rails in the container to 3000) I added some readme updates for this.